src/components/form-draft/checklist.vue

```diff
@@ -136,11 +136,11 @@ export default {
       },
       {
         // This is the title of a checklist item.
-        "title": "Upload Form Media Files",
+        "title": "Upload Form Attachments",
         "body": [
           {
-            "full": "Your Form design references files that are needed in order to present your Form. You can upload new or updated copies of these for distribution under the {mediaFiles} tab.",
-            "mediaFiles": "Media Files"
+            "full": "Your Form design references files that are needed in order to present your Form. You can upload new or updated copies of these for distribution under the {formAttachments} tab.",
```

Comment:
This text also needs to account for the possibility of datasets (the user might not need to upload anything, just link a dataset). But we can also make that change later.